pkg/scheduler/server/internal/etcd/etcd.go

```diff
@@ -126,6 +127,11 @@ func (e *etcd) Close() error {
 
 	var err error
 	if e.client != nil {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
+		defer cancel()
+		if _, err = e.client.MemberRemove(ctx, uint64(e.etcd.Server.ID())); err != nil {
```

Comment:
Won't this compromise the durability of the data on etcd? If all instances are shutdown at once?